### PR TITLE
Bump knative/pkg to master

### PR DESCRIPTION
**Pull request by vdemeester**
This is part kind of knative/pkg#97 as it updates knative/pkg to master and update the uses of `duck.VerifyType` (changed in knative/pkg#98)

cc @dprotaso @mattmoor

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -42,6 +42,11 @@ required = [
   name = "k8s.io/client-go"
   version = "kubernetes-1.11.0"
 
+[[override]]
```
Comment (shashwathi) [inline Gopkg.toml]: There are 2 versions of pkg dependency in toml file. I think you can update the one under `constraint` to recent sha.

File: pkg/apis/build/v1alpha1/build_types_test.go [hunk 2]
```diff
@@ -30,7 +30,9 @@ import (
 const bazelYAML = "testdata/cloudbuilders/bazel/cloudbuild.yaml"
 
 func TestBuildImplementsConditions(t *testing.T) {
-	duck.VerifyType(&Build{}, &duckv1alpha1.Conditions{})
+	if err := duck.VerifyType(&Build{}, &duckv1alpha1.Conditions{}); err != nil {
+		t.Error(err)
```
Comment (shashwathi) [inline pkg/apis/build/v1alpha1/build_types_test.go]: An error msg along with error would be helpful
eg: `t.Errorf("Expect Build to implement duck verify type: err %#v", err )`

File: Gopkg.toml (hunk 1 above)
Comment (vdemeester) [reply Gopkg.toml]: oh :sweat_smile: I didn't look below the `[[prune]]` options, my bad :sweat_smile:

**Pull request merged**